nautobot/extras/models/models.py

```diff
@@ -944,6 +953,187 @@ def log(
             logger.log(log_level, str(message))
 
 
+class ScheduledJobs(models.Model):
+    """Helper table for tracking updates to scheduled tasks.
+    This stores a single row with ident=1.  last_update is updated
+    via django signals whenever anything is changed in the ScheduledTask model.
+    Basically this acts like a DB data audit trigger.
+    Doing this so we also track deletions, and not just insert/update.
+    """
+
+    ident = models.SmallIntegerField(default=1, primary_key=True, unique=True)
+    last_update = models.DateTimeField(null=False)
+
+    objects = ExtendedManager()
+
+    @classmethod
+    def changed(cls, instance, **kwargs):
+        if not instance.no_changes:
+            cls.update_changed()
+
+    @classmethod
+    def update_changed(cls, **kwargs):
+        cls.objects.update_or_create(ident=1, defaults={"last_update": timezone.now()})
+
+    @classmethod
+    def last_change(cls):
+        try:
+            return cls.objects.get(ident=1).last_update
+        except cls.DoesNotExist:
+            pass
```

Comment:
Would `return None` be more explicit in its intent?

Reply:
I actually feel like `return None` is usually a bit of an antipattern, but consistency might be a more useful metric anyway. Do we have precedents for doing/not doing this in the codebase?

Reply:
I agree that the pattern of "return object or return None" is not always ideal, but that's what you're implicitly doing here already with the `pass` - my suggestion was just to replace the implied `return None` with an explicit `return None`.

Reply:
I understood that, but I feel like the explicit `return None` is somewhat looked down upon by some, while it’s idiomatic in the eyes of others. That’s why I was asking what stance Nautobot takes. I got my answer, though, and I’ll fix it :)